macros.py

```diff
@@ -831,20 +840,35 @@ def overall_graph():
                 graph_file.write(line)
         graph_file.write("}\n")
 
-def month_graphs(dates):
+def month_graphs(dates, version=None):
     """Produce a graph per month, showing the exploits that were possible on the first day of that month"""
+    version_string = version
+    if version == None:
+        version_string = 'all'
+    print("Graphing version " + version_string)
+    # Create a folder for graphs of this version, if it doesn't exist already
+    folder_path = 'graphs/' + version_string.replace('.', '-')
```

Comment:
Nothing wrong with dots in folder names.

Reply:
Updated